queries.sql

```diff
@@ -1,3 +1,8 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon%';
```

Comment:
- [ ] Kindly add a query to find all animals with a weight between 10.4kg and 17.3kg as this is also part of the requirements